### PR TITLE
OW-744 - last ui fixes

### PR DESCRIPTION
**Pull request by gmininberg**

### CODE REVIEW

File: Unity/opentok/holo/holo/ViewController.mm [hunk 1]
```diff
@@ -665,9 +670,11 @@ - (void)subscriber:(nonnull OTSubscriberKit *)subscriber rtcStatsReport:(nonnull
     stats = [NSString stringWithFormat:@"Subscriber: %@\n", [inboundStats length] > 0 ? inboundStats : @"NA"];
     dispatch_async(dispatch_get_main_queue(), ^{
         [self->_subscriberStatsLabel removeFromSuperview];
-        [self->_subscriberStatsLabel setText:stats];
-        [self->_subscriberStatsLabel setNumberOfLines:0];
-        [[[[self unityFramework] appController] rootView]addSubview:self->_subscriberStatsLabel];
+        if(self->_enableLogs){
```
Comment (joliveraortega) [inline Unity/opentok/holo/holo/ViewController.mm:673]: Should also include `[self->_subscriberStatsLabel removeFromSuperview];` here?

File: Unity/opentok/holo/holo/ViewController.mm [hunk 2]
```diff
@@ -629,9 +632,11 @@ - (void)publisher:(nonnull OTPublisherKit *)publisher rtcStatsReport:(nonnull NS
         stats = [NSString stringWithFormat:@"Publisher: %@", [outboundStats length] > 0 ? outboundStats : @"NA"];
         dispatch_async(dispatch_get_main_queue(), ^{
             [self->_publisherStatsLabel removeFromSuperview];
-            [self->_publisherStatsLabel setText:stats];
-            [self->_publisherStatsLabel setNumberOfLines:0];
-            [self->_localVideoView addSubview:self->_publisherStatsLabel];
+            if(self->_enableLogs){
```
Comment (joliveraortega) [inline Unity/opentok/holo/holo/ViewController.mm:635]: Should also include `[self->_publisherStatsLabel removeFromSuperview];` here?

File: Unity/opentok/holo/holo/ViewController.mm [hunk 3]
```diff
@@ -708,6 +713,10 @@ - (void)roomNameAndRoleNotification:(NSNotification *)notification {
         _roomName = [userInfo valueForKey:@"roomName"] ;
     }
     
+    if([userInfo objectForKey:@"enableLogs"]){
+        _roomName = [userInfo valueForKey:@"enableLogs"] ;
```
[the remark this answers is not in the document]
Comment (gmininberg) [reply Unity/opentok/holo/holo/ViewController.mm:717]: fixed

File: Unity/opentok/holo/holo/ViewController.mm (hunk 1 above)
Comment (gmininberg) [reply Unity/opentok/holo/holo/ViewController.mm:673]: i dont think its a must but we will know once we test it.

File: Unity/opentok/holo/holo/ViewController.mm (hunk 2 above)
Comment (gmininberg) [reply Unity/opentok/holo/holo/ViewController.mm:635]: same

File: Unity/opentok/holo/holo.xcodeproj/xcshareddata/xcschemes/holo.xcscheme
Comment (joliveraortega) [inline Unity/opentok/holo/holo.xcodeproj/xcshareddata/xcschemes/holo.xcscheme:1]: The name in here did not get updated. Why?

**Review by joliveraortega — APPROVED** [review]

File: Unity/opentok/holo/holo.xcodeproj/xcshareddata/xcschemes/holo.xcscheme
Comment (gmininberg) [reply Unity/opentok/holo/holo.xcodeproj/xcshareddata/xcschemes/holo.xcscheme:1]: https://stackoverflow.com/questions/238980/how-can-i-change-the-name-of-an-ios-app-in-xcode

**Pull request merged**